packages/global/components/layouts/content/webinar.marko

```diff
@@ -0,0 +1,168 @@
+import { get, getAsArray } from "@parameter1/base-cms-object-path";
+import contentIframe from "@pmmi-media-group/package-global/utils/content-iframe";
+import getContentPreview from "@parameter1/base-cms-marko-web-theme-monorail/utils/get-content-preview";
+
+$ const { site, contentGatingHandler, req, i18n } = out.global;
+$ const { id, type, pageNode, showReadNextBlock, showBottomAdBlock, showTopStoryBlock, ...rest } = input;
+$ const sections = getAsArray(input, "sections");
+
+$ const leadersContentTypes = input.leadersContentTypes || ["article", "blog", "media-gallery", "news", "podcast", "press-release", "product", "video", "webinar"];
+$ const displayComments = false; // ["contact"].includes(type) ? false : true;
+$ const continueLabel = i18n("Sign Me Up!");
+$ const loginEmailLabel = i18n("Work Email");
+$ const buttonLabels = { continue: continueLabel };
+
+<global-content-wrapper-layout
+  id=id
+  type=type
+  page-node=pageNode
+  ...rest
+>
+  <@section|{ aliases }| modifiers=["first-sm"]>
+    <theme-gam-define-display-ad
+      name="leaderboard"
+      position="content_page"
+      aliases=aliases
+      modifiers=["inter-block"]
+    />
+    <theme-gam-wallpaper-ad aliases=aliases position="content_page" />
+  </@section>
+
+  <@section|{ blockName, content, aliases, authors }| modifiers=["content-header"]>
```

Comment:
authors can be removed here along with lines 36 - 46.  I webinars do not have an author field.